src/reducer.ts

```diff
@@ -19,7 +19,8 @@ const wrapInArray = <T>(val: T | T[]): T[] =>
 
 export type Reducer<State, Payload = VoidPayload> = (
   previousState: State,
-  payload: Payload
+  payload: Payload,
+  namespace?: string
```

Comment:
I still think default namespace being null is better than optional.

Reply:
Yes. I've created an issue for that ARD-6151
However, it is out of scope for this change

Reply:
Sure, that was more a note